Makefile

```diff
@@ -22,13 +22,19 @@ ASSETS=$(shell grep -oh '[^"]*/.*\.yaml' pkg/manifests/manifests.go \
 
 BIN_DIR ?= $(shell pwd)/tmp/bin
 
+# Docgen related variables
+TYPES_TARGET=pkg/manifests/types.go
+K8S_VERSION=$(shell echo -n v1. &&  cat go.mod | grep -w "k8s.io/api" | awk '{ print $$2 }' | cut -d "." -f 2)
+PO_VERSION=$(shell cat go.mod | grep -w "github.com/prometheus-operator/prometheus-operator" | awk '{ print $$2 }' | sort -u)
```

Comment:
```suggestion
K8S_VERSION=$(shell echo -n v1. &&  grep -w "k8s.io/api" go.mod | awk '{ print $$2 }' | cut -d "." -f 2)
PO_VERSION=$(shell grep -w "github.com/prometheus-operator/prometheus-operator" go.mod | awk '{ print $$2 }' | sort -u)
```

Reply:
Nit but `grep` doesn't need `cat` for a file and less pipes make this a bit friendlier to read..?